pkg/core/blockchain_test.go

```diff
@@ -15,6 +15,8 @@ import (
 
 func TestAddHeaders(t *testing.T) {
 	bc := newTestChain(t)
+	defer bc.Close()
```

Comment:
I guess this line really belongs to 04a5d85dc83de038033af13eb4df949b4980b81d.

Reply:
Fixed.